blockchain/beacon/src/beacon_entries.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use encoding::{
+    de::{self, Deserializer},
+    ser::{self, Serializer},
+    serde_bytes,
+};
+use serde::Deserialize;
+
+/// The result from getting an entry from Drand.
+#[derive(Clone, Debug, Default, Eq, PartialEq)]
+pub struct BeaconEntry {
+    round: u64,
+    data: Vec<u8>,
+    prev_round: u64,
```

Comment:
Why have this as a variable? Not needed when prev_round can just give you this?

Reply:
It will be used later in an upcoming PR.

Reply:
but can always just call the function for this instead of having it as a field?